content/md/en/docs/maintain/monitor.md

```diff
@@ -4,59 +4,174 @@ description:
+However, you can expose it on all interfaces by using the `--prometheus-external` command-line option to start a node.
 
+### Configure monitoring tools
 
+To set up monitoring and alerting policies, you typically configure a set of tools to create your own monitoring stack.
+For example, the default `metrics` endpoint doesn't include host metrics—such as CPU, memory, bandwidth usage—so you can complement it by installing the Prometheus [node_exporter](https://github.com/prometheus/node_exporter) on each host.
+The following diagram illustrates an open source set of tools that are often used as a monitoring stack. 
 
-The telemetry information includes details about the hardware and software running on the node, including:
+![Monitoring stack layers](/media/images/docs/monitoring-stack.png)
 
-- The exact CPU model.
+As this diagram illustrates, there are different tools available for each layer of the stack.
+In this example, the following tools are configured for monitoring on-chain activity and node operations:
 
-- The total amount of memory, in bytes.
+- Prometheus is a monitoring engine that collects metrics from specified targets at specified intervals and evaluates the data collected using rules you define. Its time series database can hold large amounts of data that can be accessed very quickly.
+- Grafana is an observability platform that allows you to query, visualize, and analyze the data you collect through dashboards and graphs.
+- Node exporter is process that listens on a port and reports application-specific metrics to Prometheus.
+- Alertmanager is a tool that enables you to create and route alerts based on the rules you specify. Alertmanager allows you to configure how and where to send alert if something goes wrong. For example, you can send instant messages for warning alerts, but page an on-call technician for critical alerts
+- Loki is a scalable log aggregation system that allows you to view and search logs from all components in your infrastructure in one place.
 
-- The number of physical CPU cores.
+For a simple example of setting up node monitoring using Prometheus, Grafana, and node exporter, see [Monitor node metrics](/tutorials/get-started/monitor-node-metrics/).
+For a simplified example of using Loki, see [Remote monitoring](/deploy/deployment-options/#remote-monitoring).
 
-- The Linux kernel version.
+### Change the telemetry server
 
-- The exact Linux distribution used.
+After you have configured the backend monitoring rules for your nodes, you can use the `--telemetry-url` command-line option when you start a node to specify the telemetry server that you want to send telemetry data to.
+You can pass this option multiple times to specify multiple telemetry endpoints. 
+You can also specify how verbose metrics should be, with level 0 denoting the least verbose through level 9 denoting the most verbose. 
```

Comment:
```suggestion
You must also specify how verbose metrics should be, with level 0 denoting the least verbose through level 9 denoting the most verbose. 
```

As in; you must provide both a URl and a verbosity level for the option to be valid :)

Reply:
Ah, didn't know that!